kokoro-linux.sh

```diff
@@ -0,0 +1,20 @@
+# Install NVM
```

Comment:
The file you're adding is 644. It needs to be 755.

Reply:
I think I forget that more often than I remember.